spec/robots/assembly/checksum_compute_spec.rb

```diff
@@ -4,7 +4,8 @@
 
 RSpec.describe Robots::DorRepo::Assembly::ChecksumCompute do
   let(:robot) { described_class.new }
-  let(:druid) { 'aa222cc3333' }
+  let(:druid) { 'bb222cc3333' }
```

Comment:
For what it is worth, in other places we use `druid` and `bare druid`.

Reply:
true.  fixed.